networking/networking.md

```diff
@@ -15,7 +15,18 @@ A LinkedIn profile
 
 
 ### Motivation
-At some point in our careers, every one of us will need to ask others for feedback, connections, or support. The more you can build and manage your network, the better able you'll be to achieve your goals. With the right strategies, networking can be enjoyable and advantageous.
+At some point in our careers, every one of us will need to ask others for feedback, connections, or support. The more you can build and manage your network, the better able you'll be to achieve your goals. With the right strategies, networking can be enjoyable and advantageous. The wider the connections more is the chance to explore and express your skills. It is also important when it comes to searching for jobs. It is one of the important strategies for the advancement of one's career.
+
+Benefits of networking: 
+- Generation of referrals/Increased business
+- Opportunities
+  - With a motivated group of startup owners comes an abundance of opportunities! There are always lots of opportunities that come from networking and in fact this is where the benefits of business networking are huge!
```

Comment:
- With a motivated group of people comes an abundance of opportunities! There are always lots of opportunities that come from networking and in fact this is where the benefits of networking are huge!

Reply:
(Removed business owner context)